File: 프로그래머스/0/181909. 접미사 배열/접미사 배열.cs
using System;
using System.Collections.Generic;
public class Solution {
    public string[] solution(string my_string) {
        List<string> list = new List<string>();
        for(int i = 0 ; i < my_string.Length;i++)
        {
            list.Add(my_string.Substring(i,my_string.Length - i));
        }
        list.Sort();
        string[] answer = list.ToArray();
        return answer;
    }
}

File: 프로그래머스/0/181909. 접미사 배열/접미사 배열.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<string> solution(string my_string) {
    vector<string> answer;
    for(int i = 0; i < my_string.size(); i++)
    {
        string temp = my_string.substr(i,my_string.size() - i);
        answer.push_back(temp);
    }
    sort(answer.begin(),answer.end());
    return answer;
}